vscode-smart-contracts/scripts/linux-vscode-smart-contracts.Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+pipeline {
+    agent { label 'jenkins-worker' }
+    environment {
+        VERSION = sh(
+            returnStdout: true,
+            script: '''\
+                # Extract version number if not set as parameter
+                [ -z "$VERSION" ] && VERSION=$(node -p "require('vscode-smart-contracts/package.json').version")
+                echo -n "$VERSION"
+            '''.stripIndent()
+        )
+        CARGO_CONCORDIUM_VERSION = "${VERSION}"
+        OUTFILE = "s3://distribution.concordium.software/tools/linux/vscode-smart-contracts_${VERSION}"
```

Comment:
I'm assuming this is not just for Linux, so we could change that to vscode or plugins or something.

Reply:
nevermind, saw the other comments

Reply:
There will be a package for each platform as it contains the `cargo-concordium` binary.
VS code marketplace supports platform specific builds